<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="NAUSP-ELPT82228" failures="1" tests="6" name="Test1" time="0.211" errors="0" timestamp="2020-06-20T16:18:10 EDT">
  <testcase classname="test.TestNGRerunFailedTestCase" name="test1" time="0.019"/>
  <testcase classname="test.TestNGRerunFailedTestCase" name="test2" time="0.005"/>
  <testcase classname="test.TestNGRerunFailedTestCase" name="test3" time="0.01">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="test.TestNGRerunFailedTestCase" name="test3" time="0.005">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="test.TestNGRerunFailedTestCase" name="test3" time="0.005">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="test.TestNGRerunFailedTestCase" name="test3" time="0.007">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.TestNGRerunFailedTestCase.test3(TestNGRerunFailedTestCase.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
</testsuite> <!-- Test1 -->
